store/review.ts

```diff
@@ -50,8 +53,11 @@ export const useReviewStore = create<ReviewStore>((set: any, get: any) => ({
   setActiveBadges: (activeBadges: Badge[] | null) =>
     set((state: any) => ({ ...state, activeBadges })),
   activeBadgeIds: null,
-  setActiveBadgeIds: (activeBadgeIds: Badge[] | null) =>
+  setActiveBadgeIds: (activeBadgeIds: Hex[] | null) =>
```

Comment:
@heronlancellot mentioned this was an error he found during this branch's work.

Reply:
@heronlancellot has discovered this as an issue in build, I have included this as a hotfix to make gap-app-v2 build possible, in another [branch](https://github.com/blockful-io/gap-app-v2/pull/19)